integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/cloud/AwsCloudProvider.java

```diff
@@ -91,6 +90,33 @@ public CredentialEntity aValidCredential(boolean create) {
         return map;
     }
 
+    public Map<String, Object> awsCredentialDetailsKey() {
+        Map<String, Object> map = new HashMap<>();
+        map.put("selector", "key-based");
+        map.put("accessKey", getTestParameter().get("integrationtest.awscredential.accessKey"));
+        map.put("secretKey", getTestParameter().get("integrationtest.awscredential.secretKey"));
```

Comment:
just a suggestion :)
In Java 10 you can simply create a new map with the Map.of(...) call. This specific (and the other) methods could be simplified to something like this:

`return Map.of("selector", "key-based", 
                "accessKey", getTestParameter().get("integrationtest.awscredential.accessKey"), 
                "secretKey", getTestParameter().get("integrationtest.awscredential.secretKey"));`

this comment/suggestion relates to all Map related changes in the commit. Not a request, just a hint.